code-challenges.test.js

```diff
@@ -27,41 +27,88 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
+// describe ("divByThree", () => {
+//     it("returns a string letting you know if the entered number is devisible by 3 or not.", () => { 
+//         const object1 = { number: 15 }
+//         const object2 = { number: 0 }
+//         const object3 = { number: -7 } 
+//     expect(divByThree(object1.number)).toEqual(`${object1.number} is divisible by three`)
+//     expect(divByThree(object3.number)).toEqual(`${object3.number} is not divisible by three`)
```

Comment:
The prompt specifies that the function will take an object as an argument. Here, you are giving the function a number as an argument